app/scripts/modules/core/src/application/application.less

```diff
@@ -1,5 +1,11 @@
 @import (reference) "~core/presentation/less/imports/commonImports.less";
 
+application {
```

Comment:
We will need to migrate this to a class when we switch to react.  Since this is new styling, maybe we should use a class now